### DISCUSSION TITLE
Garmin Edge 830 & Varia RTL 515 Error 04

### DISCUSSION THREAD

**Discussion by Alma905** [maca88/SmartBikeLights#56] [General]
All attempts to transfer a configuration to the Edge 830 lead to the error message "Error04" For example, I have tried the variant that the light only turns on and flashes when a car approaches and then goes out again.
For this I have created the following configuration: ###0,73535488::1#2,2! Carbon:1:4:0:0I[300]0!:1:0:0:0D=1##6,5:Varia 515:0:16777215! 2,:-1,Off:0! 1,Solid:4! 1,Peloton:5! 1,Day Flash:7! 1,Night Flash:6#0::#0:1#123!:123! #0#0#B3122##3#0#0
What is wrong ?

Greeting

**Comment by maca88** [comment]
Hi, sorry for the late reply. Your configuration for some reason contains whitespaces, which makes it invalid. Just remove the whitespace and it should work:

###0,73535488::1#2,2!Carbon:1:4:0:0I[300]0!:1:0:0:0D=1##6,5:Varia 515:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:1#123!:123!#0#0#B3122##3#0#0

**Reply by Alma905** [reply]
Hello
Thank you for your help.
I can Not See any whitespaces in the „Sting“. 
Thany you
